tests/stack/test_cli.py

```diff
@@ -0,0 +1,54 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint:disable=redefined-outer-name
+
+import pytest
+import mock
+from click.testing import CliRunner
+
+import databricks_cli.stack.cli as cli
+from tests.utils import provide_conf
+
+
+@pytest.fixture()
+def stack_api_mock():
+    with mock.patch('databricks_cli.stack.cli.StackApi') as StackApiMock:
+        stack_api_mock = mock.MagicMock()
+        StackApiMock.return_value = stack_api_mock
+        yield stack_api_mock
+
+
+@provide_conf
+def test_deploy(stack_api_mock, tmpdir):
```

Comment:
We should have a test which deploys a stack and validates the deployed JSON which is written back out. Feel free to push to a different PR.

Reply:
sounds good, I'll add a TODO in the file.